### ISSUE TITLE
NPE while connecting to Multiplayer Server

### ISSUE THREAD

**Issue by Kilobyte22** [fnuecke/oc2#48]
When trying to play with oc2 on a test server i got the following NPE:

```
[22:00:12] [Server thread/INFO] [minecraft/ServerPlayNetHandler]: gruetzkopf lost connection: Internal Exception: io.netty.handler.codec.EncoderException: java.lang.NullPointerException
```

This happens with both dedicated Server and LAN Game on Forge 36.1.0. The Error happens on the server, but no stack trace is printed. The Error does not happen when i remove the `OpenComputers-II-*-Forge*` file on both server and client (only removing it on the server leads to forge complaining about mod incompatibilities)

**Comment by sd-fritze** [comment]
I've briefly looked into this, NPE occurs in the WRENCH receipe serialiser, not registering it leads to being able to connect.

**Comment by dequbed** [comment]
It appears to be a version fuckery with the builds using artifacts from maven. I changed gradle up to use the github repos and generate jars from that and the NPE we had vanished.

**Comment by CoolCat467** [comment]
That's good to hear.

**Comment by atirutw** [comment]
> It appears to be a version fuckery with the builds using artifacts from maven. I changed gradle up to use the github repos and generate jars from that and the NPE we had vanished.

Doesn't that make this issue solved, then?

**Comment by fnuecke** [comment]
Yeah, should've been a broken recipe registration iirc, should be fixed as of 68205bf759b59529e9800632071df23977882685.
